device-types/Datacom/DM4380-12XS-plus-3CX.yaml

```diff
@@ -0,0 +1,64 @@
+---
+manufacturer: Datacom
+model: DM4380 12XS+3CX
+slug: datacom-dm4380-12xs-plus-3cx
+part_number: 800.5258.xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: Managed L2, L3, and MPLS switch. MPLS functionality requires a separately purchased license. Includes 12 1/10GbE ports on SFP+, 3 40/100GbE ports
+  on QSFP28, outband management interfaces, RS232 console, and USB ports. SFPs, SFP+, QSFP+, and QSFP28 modules purchased separately. 1U metal chassis for
+  19 rack. AC or DC power supply with redundancy option (purchased separately).
+weight: 3
+weight_unit: kg
+console-ports:
+  - name: Console0
+    type: rj-45
+  - name: Console1
+    type: usb-mini-b
+console-server-ports:
+  - name: Storage
+    type: usb-a
```

Comment:
Is this actually used as a console server port? Or is it just for the switch to access a USB storage device? If the latter, please remove this definition as it would be out of scope.

Reply:
removed